features/uuri/pattern_matching.feature

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-FileType: SOURCE
+# SPDX-License-Identifier: Apache-2.0
+#
+Feature: Matching endpoint identifiers (UUri) against patterns
+
+  Scenario Outline:
+    Developers using a uProtocol language library should be able to verify that a specific
+    endpoint identifier matches a given pattern as specified by the UUri specification.
```

Comment:
Is it possible to add links (to the spec) here? Probably something to do once we move this over to up-spec...

Reply:
Instead of linking directly to the corresponding sections in the spec documents, I would rather rely on the OFT specification items which are used to link to the corresponding requirements defined in the specification documents. Links tend to become invalid over time without anybody noticing. However, if the OFT spec items are changed, we will immediately know when doing the trace run ...